EMIP/3001-4000/EMIP03003.xml

```diff
@@ -28,22 +28,258 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.16">
+<locus from="43r" to="49r" facs="042"/>
+<title type="complete" ref="LIT2149OntheW"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.17">
+<locus from="49r" to="51v" facs="048"/>
+<title type="complete" ref="LIT6441Kehda"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.18">
+<locus from="51v" to="55r" facs="051"/>
+<title type="complete" ref="LIT6444HomilyPilatos"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.19">
+<locus from="55r" to="60v" facs="054"/>
+<title type="incomplete" ref="LIT1750LahaMa"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.20">
+<locus from="60v" to="66r" facs="060"/>
+<title type="complete" ref="LIT2118OntheD"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.21">
+<locus from="66r" to="71v" facs="065"/>
+<title type="complete" ref="LIT5862OnSpiritualEaster"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.22">
+<locus from="71v" to="74v" facs="071"/>
+<title type="complete" ref="LIT2109OntheA"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.23">
+<locus from="74v" to="80r" facs="074"/>
+<title type="complete" ref="LIT2137OntheP"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.24">
+<locus from="80r" to="82r" facs="079"/>
+<title type="complete" ref="LIT6047On12ApostlesRetuaH"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.25">
+<locus from="82r" to="85r" facs="081"/>
+<title type="complete" ref="LIT2145OntheT"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.26">
+<locus from="85r" to="92v" facs="084"/>
+<title type="complete" ref="LIT1640Homily"></title>
+<textLang xml:lang="gez"/>
+</msItem>
+
+</msItem>
+
                     </msContents>
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
-                                
+                              <support>
+                                 <watermark/>
+                                 <material key="parchment"/>
+                              </support>
+                              <extent>
+                                 <measure unit="leaf">93</measure>
+                                 <dimensions type="outer" unit="mm">
+                                    <height>345</height>
+                                    <width>250</width>
+                                    <depth/>
+                                 </dimensions>
+                              </extent>
+
+<collation>
+  <signatures>
+     <note>Quires are marked with Ethiopic numerals, on the upper inner corner of the first leaf of each quire.</note>
+  </signatures>
+
+<list>
+  <item xml:id="q1" n="A"><dim unit="leaf"></dim>I-III<note>F. 1r not microfilmed.</note></item>
```

Comment:
the dim element remained empty, producing an error https://github.com/BetaMasaheft/Documentation/issues/2295